### ISSUE TITLE
Warning during install with `storybook/addons` ^4.0.0-alpha.25 [0.3.4]

### ISSUE THREAD

**Issue by PeterMK85** [gvaldambrini/storybook-router#29]
Issue:

### Which version are you using?
`storybook-router@0.3.4`

### Are you using storybook-router with a react based project or a vue one?
React

### Please describe the problem:
During install warning displayed:
`"storybook-router > @storybook/addon-actions@3.4.7" has incorrect peer dependency "@storybook/addons@^3.3.0".
`

### Please explain how to reproduce the issue or (better) provide an example to do it.
After Webpack 4 and Babel 7 update, storybook update was also necessary but it seems that the `storybook-router` give a warning on this one

Version:
```
"@storybook/addons": "^4.0.0-alpha.25",
```

**Comment by gvaldambrini** [comment]
Sorry for the delay, have you tried with the package `storybook-react-router`? `storybook-router` was the old version which includes both the support for React and Vue. Please upgrade your dependency.

**Comment by csantos1113** [comment]
Same:
```
warning " > storybook-react-router@1.0.1" has incorrect peer dependency "@storybook/addon-actions@^3.0.0".
warning " > storybook-react-router@1.0.1" has incorrect peer dependency "@storybook/addons@^3.0.0".
```
my package.json
```
"@storybook/addon-actions": "^4.0.2",
"storybook-react-router": "^1.0.1"
```

**Comment by arty-name** [comment]
I am seeing the same thing: storybook-react-router depends on storybook v3

**Comment by arty-name** [comment]
[v3 is actually explicitly defined in the package.json](https://github.com/gvaldambrini/storybook-router/blob/f5e650794ebbb02d73c8e62c7813ae84e400637b/packages/react/package.json#L22-L23)

**Comment by gvaldambrini** [comment]
The new published 1.0.2 supports storybook 4.X. Closing this
